src/registrar/config/settings.py

```diff
@@ -297,12 +297,16 @@
 
 # Content-Security-Policy configuration
 # this can be restrictive because we have few external scripts
-allowed_sources = ("'self'",)
+allowed_sources = "'self'"
+allowed_sources_scripts = ["'self'", "https://www.googletagmanager.com/", "https://www.google-analytics.com/"]
```

Comment:
**suggestion (non-blocking)**: We never use this `allowed_sources_scripts` variable now, so I think you can remove this line.

Reply:
oh yes, good catch